cwe_api/cvss_inferencer.py

```
@@ -0,0 +1,145 @@
+        if index != 7:
+            cvss_string += '/'
+
+    return cvss_string
+
+
+def inferencer(vuln):
+
+    set_global_logging_level(logging.ERROR, ["transformers", "nlp", "torch", "tensorflow", "tensorboard", "wandb"])
+
+    with open(NUM_LABELS_PATH, 'rb') as handle:
+        num_labels = pickle.load(handle)
```

Ruff

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)